ingestion_server/ingestion_server/cleanup.py

```diff
@@ -351,16 +351,15 @@ def clean_image_data(table):
                         cleanable_fields_for_table,
                     )
                 )
-            pool = multiprocessing.Pool(processes=num_workers)
-            log.info(f"Starting {len(jobs)} cleaning jobs")
-
-            results = pool.starmap(_clean_data_worker, jobs)
-
-            for result in results:
-                batch_cleaned_counts = save_cleaned_data(result)
-                for field in batch_cleaned_counts:
-                    cleaned_counts_by_field[field] += batch_cleaned_counts[field]
-            pool.close()
+            with multiprocessing.Pool(processes=num_workers) as pool:
+                log.info(f"Starting {len(jobs)} cleaning jobs")
+
+                for result in pool.starmap(_clean_data_worker, jobs):
```

Comment:
Since `pool.starmap` is blocking (like `pool.map`), as per this [StackOverflow answer](https://stackoverflow.com/a/55035444) I don't think the `close()` and `join()` should be needed at the end.

Reply:
Without the calls to `close()` and `join()`, the ingestion test was failing in CI. Probably because the audio ingestion was throwing an error about the index already existing.